README.rst

```diff
@@ -1,130 +1,154 @@
-`our capabilities listing <https://github.com/coala-analyzer/bear-docs/blob/master/README.rst>`__.
+.. code-block:: bash
 
-*coala* is written with a lower case "c".
+    $ pip3 install coala-bears
 
-Read more at our `documentation <http://coala.rtfd.org/>`__.
+To install the latest development version run:
 
-Why use coala?
---------------
+.. code-block:: bash
 
-- *coala* provides **built-in checking routines** (named bears in *coala*).
-- **Serves your requirement**: You can easily write your own checks (using
-  bears).
-- *coala* provides **user-friendly interfaces** like json, formatted and
-  interactive output in the CLI and *plugins for various editors* are
-  available as well.
-- **Optimal performance**: *coala* manages parallelizing the checking-routines
-  without you having to worry.
-- **Unified interface**: One tool for all programming languages.
+    $ pip3 install coala-bears --pre
 
-If you want to learn more about *coala*, its functionality and its usage,
-please take a look at our
-`tutorial <http://coala.rtfd.org/en/latest/Users/Tutorials/Tutorial.html>`__.
+|PyPI| |Windows| |Linux|
 
-Installation
-------------
+-----
 
-To install *coala* with the official set of analysis routines you can simply run
-``pip3 install coala-bears``.
+=====
+Usage
+=====
 
-*coala* only, without the official bears, can be installed with
-``pip3 install coala``. If you need more information about the installation and
-dependencies, take a look at our `installation documentation
-<http://coala.rtfd.org/en/latest/Users/Install.html>`__.
+To get the list of arguments and their meaning, run:
 
-The latest code from master is automatically deployed to PyPI as a
-development version. Get it with ``pip3 install coala --pre``.
+.. code-block:: bash
 
-|PyPI|
+    $ coala --help
 
-Authors
--------
+To run coala over your project, create a ``.coafile``, the project-specific
+configuration file that will store all your settings for coala. A sample
+``.coafile`` will look something like this:
 
-*coala* is maintained by a growing community. Please take a look at the
-meta information in `setup.py <setup.py>`__ for current maintainers.
+.. code-block:: bash
 
-Project Status
---------------
+    [Default]
+    files = src/**/*.py
+    bears = SpaceConsistencyBear
+    use_spaces = True
 
-|Linux Build Status| |Windows Build status|
+Store this file in the project's root directory and run coala:
 
-|Scrutinizer Code Quality| |codecov.io|
+.. code-block:: bash
 
-|Documentation Status| |Gitmate|
+    $ coala
 
-Newcomers Guide and Getting Involved
-------------------------------------
+Please read our
+`coafile specification <http://coala.readthedocs.io/en/latest/Users/coafile.html>`__
+to learn more.
 
-If you are new and would like to contribute, read our `Getting Involved Site
-<http://coala.readthedocs.org/en/latest/Getting_Involved/README.html>`__!
+If you want to quickly want to test some settings without a ``.coafile``, you
```

Comment:
you've used ``want`` twice :D 

Hm i was thinking maybe something like this instead:

``If you want to quickly test some settings, without creating a ".coafile", you can also run coala in the command line and pass it arguments, such as:``